clients/include/testing_gemm_strided_batched.hpp

```diff
@@ -229,25 +211,25 @@ rocblas_status testing_gemm_strided_batched(Arguments argus)
                                                             stride_c,
                                                             batch_count));
 
-        CHECK_HIP_ERROR(hipMemcpy(hC_2.data(), dC, sizeof(T) * size_c, hipMemcpyDeviceToHost));
+        CHECK_HIP_ERROR(hipMemcpy(hC_2, dC, sizeof(T) * size_c, hipMemcpyDeviceToHost));
 
         // CPU BLAS
         cpu_time_used = get_time_us();
         for(rocblas_int i = 0; i < batch_count; i++)
         {
```

Comment:
No need for `.data()`. But must specify both template arguments `cblas_gemm<T,T>` so that all function arguments are coerced correctly.